client/src/views/asset-list.js

```diff
@@ -1,12 +1,37 @@
 import Snabbdom from 'snabbdom-pragma'
-
+import { formatAssetAmount } from './util'
 import layout from './layout'
 
-export default ({ assetMap, t, ...S }) => {
+const staticRoot = process.env.STATIC_ROOT || ''
+export default ({ assetMap, assetList, loading, t, ...S }) => {
+
+  const { assets, query, total } = !assetList ? {} : assetList
+  const { start_index, sort_dir, sort_field, limit } = !query ? {} : query
+  const pageLink = `/assets/registry?sort_field=${encodeURIComponent(sort_field)}&sort_dir=${encodeURIComponent(sort_dir)}`
+  
+  const getSupply = (asset) => {
+    let { chain_stats = {}, mempool_stats = {} } = asset
+    let has_blinded_issuances =
+        chain_stats.has_blinded_issuances || mempool_stats.has_blinded_issuances
+    let is_native_asset = !asset.issuance_txin
+    let circulating = is_native_asset
```

Comment:
Maybe worth extracting the circulating amount calculation to `util.js`?

Reply:
changes have been made